docs/recording/index.md

```diff
@@ -1,5 +1,11 @@
 # Recording with CARET
 
+## Introduction
+
+Recording includes only two steps; building an target application from source code and executing the target application. After execution, you will find trace data which has recorded events captured by trace points.
+
+If trace data is not recorded due to any reason, you will not able to analyze the target application as expected. In order to avoid it, CARET serves function to validate trace data.
```

Comment:
```suggestion
If trace data is not recorded due to any reason, you will not be able to analyze the target application as expected. In order to avoid it, CARET serves function to validate trace data.
```

Reply:
Fixed at https://github.com/tier4/CARET_doc/pull/134/commits/d42d616140310f79cdca9c54a1c58df265b492b2
Thanks,